src/js/services/conversation-service.js

```diff
@@ -0,0 +1,76 @@
+import { store } from 'stores/app-store';
+import { messageAdded, setConversation } from 'actions/conversation-actions';
+import { setFayeSubscription, unsetFayeSubscription } from 'actions/faye-actions';
+import { core } from 'services/core';
+import { immediateUpdate } from 'services/user-service';
+import { initFaye } from 'utils/faye';
+
+export function sendMessage(text) {
+    var sendFn = () => {
+        const message = {
+            text: text,
+            role: 'appUser'
+        };
+
+        store.dispatch(messageAdded(message));
```

Comment:
So, right now we add the message before making the request to the server? What happens if the request to add a message to the conversation is rejected?